rosdep/python.yaml

```diff
@@ -6129,6 +6129,9 @@ python3-pymodbus:
   ubuntu:
     '*': [python3-pymodbus]
     xenial: null
+python3-pyosmium:
+  debian: [python3-pyosmium]
+  ubuntu: [python3-pyosmium]
```

Comment:
Let's add a rule for Fedora as well: https://src.fedoraproject.org/rpms/pyosmium

Reply:
I added fedora to the Python2 key `pyosmium` as well. I am not as familiar with Fedora as I am with Debian/Ubuntu, but the link you provided looks like the package name is the same for Python 2/3 in Fedora.

Reply:
Fedora 32 [dropped all Python 2 packages](https://fedoraproject.org/wiki/Changes/RetirePython2). I don't know if package `pyosmium` is compatible with Python 2 and Python 3, so maybe it's better to remove the fedora key for the Python 2 rosdep key.

@cottsay Maybe you'd like to chime in?

Reply:
@cottsay Friendly ping.

Reply:
@cottsay friendly ping

Reply:
Sorry - I didn't get any notifications for this thread for some reason.

The source package is called `pyosmium`, but the python 3 subpackage is called [`python3-osmium`](https://src.fedoraproject.org/rpms/pyosmium/blob/master/f/pyosmium.spec#_31). There is no python 2 package in any currently supported Fedora platform.